packages/ripple-nuxt-tide/README.md

```diff
@@ -2,7 +2,180 @@
 
 > The Nuxt module for adding Tide integration.
 
-## Setup
+## Nuxt configuration
+
+A `nuxt.config.js` file should exist in the root of your nuxt-tide site.
```

Comment:
`nuxt.config.js` will be in the nuxt project root by default. So I feel this line is unnecessary. Here is the setup doc template in Nuxt community you may can reference https://github.com/nuxt-community/module-template/blob/master/template/README.md

Reply:
Removed.